messaging/types/channel.perms.gen.go

```diff
@@ -64,13 +64,13 @@ func (*Channel) Permissions() []rules.OperationGroup {
 
 func (*Channel) PermissionDefault(key string) rules.Access {
 	values := map[string]rules.Access{
+		"text.embed":      rules.Inherit,
```

Comment:
please change all this to reflect names we're already using:
text => message
edit => update

plus, it all needs to make more sense (manage vs text/message vs audit vs admin)

Reply:
Did the change, please check if you have any other comments on wording.